pennylane/devices/default_tensor.py

```diff
@@ -156,15 +160,43 @@ class DefaultTensor(Device):
             statistics like expectation values and variances is performed analytically.
         method (str): Supported method. Currently, only ``"mps"`` is supported.
         dtype (type): Datatype for the tensor representation. Must be one of ``np.complex64`` or ``np.complex128``.
-            Default is ``np.complex128``.
-        **kwargs: keyword arguments. The following options are currently supported:
-
-            ``max_bond_dim`` (int): Maximum bond dimension for the MPS simulator.
-                It corresponds to the number of Schmidt coefficients retained at the end of the SVD algorithm when applying gates. Default is ``None``.
-            ``cutoff`` (float): Truncation threshold for the Schmidt coefficients in a MPS simulator. Default is ``np.finfo(dtype).eps``.
-            ``contract`` (str): The contraction method for applying gates. It can be either ``auto-mps`` or ``nonlocal``.
-                ``nonlocal`` turns each gate into a MPO and applies it directly to the MPS, while ``auto-mps`` swaps nonlocal qubits in 2-qubit gates to be next
-                    to each other before applying the gate, then swaps them back. Default is ``auto-mps``.
+        **kwargs: keyword arguments for the device, passed to the ``quimb`` backend.
+
+    Keyword Args:
+        max_bond_dim (int): Maximum bond dimension for the MPS method.
+            It corresponds to the number of Schmidt coefficients retained at the end of the SVD algorithm when applying gates. Default is ``None``.
+        cutoff (float): Truncation threshold for the Schmidt coefficients in the MPS method. Default is ``np.finfo(dtype).eps``.
```

Comment:
Does `quimb` use `cutoff` as the name here? I found it confusing because I think of `max_bond_dim` as another type of (integer-based) cutoff.

Reply:
> I found it confusing because I think of max_bond_dim as another type of (integer-based) cutoff.
I thought the same. It is actually the same name used in the [`quimb` documentation](https://quimb.readthedocs.io/en/latest/autoapi/quimb/tensor/circuit/index.html#quimb.tensor.circuit.CircuitMPS).

By the way, I see that now the parameters like `cutoff`, `max_bond_dim`, etc. can be passed directly to `CircuitMPS`, while in the previous version it was necessary to pass them using the `apply_gate` function. Nice!